js/compiler.ts

```diff
@@ -123,8 +131,10 @@ function jsonEsmTemplate(
   jsonString: string,
   sourceFileName: string
 ): OutputCode {
-  // tslint:disable-next-line:max-line-length
-  return `const _json = JSON.parse(\`${jsonString}\`)\nexport default _json;\n//# sourceURL=${sourceFileName}`;
+  return (
+    `const _json = JSON.parse(\`${jsonString}\`)\nexport default _json;\n` +
```

Comment:
Split the string literal at the other line break

Reply:
nit: Missing semicolon after `JSON.parse(...)`.

Reply:
Fixed both.